packages/charting/src/components/BenchmarkBarChart/BenchmarkBarChart.types.ts

```diff
@@ -0,0 +1,125 @@
+import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+import { IStyleFunctionOrObject } from 'office-ui-fabric-react/lib/Utilities';
+
+export interface IBenchmarkBarChartProps {
+  /**
+   * Chart data point for the benchmark bar chart
+   */
+  data: IChartingProps;
+
+  /**
+   * This property tells whether to show benchmark triangle on top of benchmark bar chart or not
+   * @default false
+   */
+  hideBenchmark?: boolean;
+
+  /**
+   * This property tells whether to show pencentage or fraction on top right of benchmark bar chart
+   * @default false
+   */
+  isPercentage?: boolean;
+
+  /**
+   * Height of bar chart
+   * @default 10
+   */
+  height?: number;
+
+  /**
+   * Width of bar chart, it could be number or percentage
+   * For example: 10, 10%
+   * @default 300
+   */
+  width?: number | string;
```

Comment:
> width?: number | string; [](start = 1, length = 25)

why don't we just give one option that is number and internally convert that to %?

Reply:
In this scenario, number and string are different. Number=10 means 10px, string="10%" means 10 percent. I'll add more detailed comment on it.